### PR TITLE
style edit, party missing

### PR DESCRIPTION
**Pull request by KnowOne134**
rest of party coundnt enter instance correctly, distance check is completed in OnEventUpdate

### CODE REVIEW

File: scripts/zones/Arrapago_Reef/npcs/_jic.lua [hunk 1]
```diff
@@ -81,10 +83,9 @@ function onInstanceCreated(player,target,instance)
         player:delKeyItem(dsp.ki.ILRUSI_ASSAULT_ORDERS)
         player:delKeyItem(dsp.ki.ASSAULT_ARMBAND)
 
-        local party = player:getParty()
         if (party ~= nil) then
             for i,v in ipairs(party) do
-                if v:getID() ~= player:getID() and v:getZoneID() == player:getZoneID() then
```
[the remark this answers is not in the document]
Comment (KnowOne134) [reply scripts/zones/Arrapago_Reef/npcs/_jic.lua]: If i leave it in only the person starting the assault is transported and rest of party is stuck at runic. Remove that section of the line and they all move to the assault

**Comment by teschnei** [comment]
Hmm.. I don't see how that would do that, but we have to leave it in
because it shouldn't be checking people in other zones

More investigation needed

**Comment by KnowOne134** [comment]
Ill try adding global zones and add , and v:getZone == dsp.zone.ARRAPAGO_REEF then

Then if that works can add the correct zones to all the other runics

**Comment by teschnei** [comment]
I would be really interested to see what the getZoneIDs were returning if
they were really making the party members unable to enter

**Comment by KnowOne134** [comment]
added this to the script
printf("%s %s ", v:getZone(), player:getZone())
output on trying to start the assault was
CZone (08E4FAF0) CZone (08E4FBB0)
Czone (08E50540) CZone (08E506C0)

change it to does work
if v:getID() ~= player:getID() and v:getZoneID() == player:getZoneID() then
or i can do the zone global way and add the zone names, up to you guys which direction to go and i can update my pull

**Comment by TeoTwawki** [comment]
```
added this to the script
printf("%s %s ", v:getZone(), player:getZone())
```
getZone and getZoneID are not the same thing.

**Pull request merged**